iosApp/Survey/Sources/Supports/Extensions/SwiftUI/CGFloat+Extensions.swift

```diff
@@ -11,5 +11,6 @@ import SwiftUI
 extension CGFloat {
 
     static var itemSpacing: CGFloat = 20.0
+    static var surveyPadding: CGFloat = 20.0
     static var defaultPadding: CGFloat = 24.0
```

Comment:
Should we make use of large/medium/low prefixes for defining a wider range of generic padding values and easier to expand later on? This way it won't be bounded to the `survey` screen only 🤔

Reply:
Fixed in https://github.com/blyscuit/kmm-swiftUI-IC/pull/59/commits/990883954214dc410b91eae965fe397e896990d8